### ISSUE TITLE
naming things

### ISSUE THREAD

**Issue by smeijer** [smeijer/testing-playground#25]
I'm working on the embed mode, and realize that this is the time that we need to have proper names for the four areas that we have.

This is my current thought. When we talk about the area's or tabs, a `js-tab` and `html-tab` make sense. I was thinking about `preview-tab` and `result-tab` for the others. 

I've also considered `advise-tab`, but as it's also showing errors, and just field information, that wouldn't be very accurate.

![image](https://user-images.githubusercontent.com/1196524/82828695-f4b31b00-9eb1-11ea-810c-cfd464a39e3f.png)

@marcosvega91, @timdeschryver, what do you think?

**Comment by marcosvega91** [comment]
hey @smeijer. Thanks for asking me an opinion about this project 🥳 .

I agree with preview and result, but I propose `input` and `query` for `html` and `js`.
I propose to use `area` as suffix like `preview-area`. Look at the image below taken from

![image](https://user-images.githubusercontent.com/5365582/82829414-7c4d5980-9eb3-11ea-953a-388e4bdc38d7.png)

**Comment by timdeschryver** [comment]
I think that's a great idea! 
(Personally, I was a bit confused with the advise element as it was also used for the error handling, so I really like these name changes).

I also like the proposals made by @marcosvega91 , I have nothing to add here 😁

Perhaps instead of `input` or `html`, what about `markup`?
Either way, I'm happy with the changes and I don't have a strong opinion, it's definite a step forward!

**Comment by marcosvega91** [comment]
I like `markup` 👍

**Comment by smeijer** [comment]
Awesome. Thanks for the suggestions. I like them :)

To recap, the areas will be named: `markup`, `preview`, `query`, and `result`.?

![image](https://user-images.githubusercontent.com/1196524/82830875-31354580-9eb7-11ea-9a90-416c8843885e.png)

**Comment by smeijer** [comment]
Do you guys also have suggestions if we divide this further in some sub-areas?

I was thinking:

```
markup

preview

query
  query editor
  query result

result
  result advise
  result queries
```

![image](https://user-images.githubusercontent.com/1196524/82834215-bec96300-9ec0-11ea-933a-64f4c769c66d.png)


An alternative that I've considered:

```
QueryResult -> QueryReturnValue
ResultAdvise -> Advise
ResultQueries -> Queries
```

![image](https://user-images.githubusercontent.com/1196524/82834350-22ec2700-9ec1-11ea-9da3-691c5283e18d.png)

**Comment by marcosvega91** [comment]
I think query editor, query result, suggestion instead of advice and queries :)

**Comment by smeijer** [comment]
I only don't like the `query result`, as I find it quite confusing with `result`. That area is the result of the query as well.

`query output` maybe?

**Comment by marcosvega91** [comment]
yes you are right. I didn't think about it

**Comment by timdeschryver** [comment]
👍 for query output.

**Comment by smeijer** [comment]
Thanks for the input guys :) I did a refactoring based on the discussion here. Result will be merged via #26.
